### DISCUSSION TITLE
Values of motors' parameters

### DISCUSSION THREAD

**Discussion by scoleli** [DLR-VSDC/IEEE-MVC-2023#33] [Q&A]
Hi,
can you provide to the participants the values of motors' parameters?
I am referring to the machine parameters in the paper equations, i.e. resistances, inductances, coefficients to compute losses.
We don't know how to open the Modelica model and look for these values and we didn't find them in the documentation so far.

Elia (_EDLab Sharks_)

**Comment by JBRDLR** [comment]
Dear Elia,

please have a look in this discussion:
https://github.com/DLR-VSDC/IEEE-MVC-2023/discussions/3#discussioncomment-4424362

As a hint where you might find them:
![image](https://user-images.githubusercontent.com/110536277/211305443-5a50aadd-f996-4f7e-9bfb-f2d4f58d6b4b.png)

Best,

Jonathan

**Comment by agubucciarelli** [comment]
Hi, after some simulations and tries, I noticied that the power losses measured from electricMotorBusFM.Ploss is not equal to the calculated by eq:
Ploss=Pinv+Pcu+Piron++Pfric
Pcu=1.5*Iq^2*rs;
Piron=0.095*(wm)+0.00009*(wm)^2;
Pinv=95+5*Iq;
Pfric=0.155*wm;

being: 
Iq=electricMotorBusFM.I_q
wm=electricMotorBusFM.angVel
for a given instant of time.

I have a smaller value with the equation. Do you know where could be the problem or what am I missing?

Thanks!

**Comment by tobolar** [comment]
@agubucciarelli I have opened a new issue to separate the topics: #16
